app/app/components/MemorableLines.jsx

```diff
@@ -0,0 +1,48 @@
+// app/components/MemorableLines.jsx
+import React from "react";
+import Image from "next/image"; // Import Image from Next.js
+import styles from "./MemorableLines.module.css"; // Import the CSS module
+
+const MemorableLines = () => {
+    return (
+        <section className={styles.memorableLines}>
+            <h2>Memorable Lines from Literature</h2>
+            <div className={styles.linesContainer}>
+                <div className={styles.line}>
+                    <div className={styles.blockquoteContainer}>
+                        <Image
+                            src="/book-icon.png"
+                            alt="The Little Prince"
+                            className={styles.quoteImage}
+                            width={100}
+                            height={100}
+                        />
+                        <h6 className={styles.blockquote}>
```

Comment:
You should always follow the heading structure on the page:
`h1 > h2 > h3 > h4 > .....`
If you want to visually change it, you need to use CSS to do that (font-size/font-weight).